requirements-tests.txt

```diff
@@ -15,6 +15,6 @@ pyyaml==6.0
 termcolor>=2
 tomli==2.0.1
 tomlkit==0.11.6
-types-pyyaml
+types-pyyaml>=6.0.12.7
```

Comment:
I can do a single PR for this if you prefer. As a followup to https://github.com/python/typeshed/pull/9752

Reply:
Fine to do it as part of this PR imo.